src/modules/Layout/Row/Row.scss

```diff
@@ -1,5 +1,20 @@
 .j-row {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+    display: flex;
+    width: 100%;
```

Comment:
@Vxplus yeap, you are right, thanks! By default `flex-grow` is one, what is exactly what I need.